modules/ROOT/pages/manage-client-apps-connected-apps-task.adoc

```diff
@@ -0,0 +1,44 @@
+
+= Creating Client Applications with Connected Apps
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: grant access, api, contracts
+
+
+For automation use cases, you can use Connected Apps to create and manage client applications.
+
+Connected Apps help user delegate their access without sharing sensitive credentials or giving full control of their applications to third parties. Actions taken by connected apps are audited, and users can revoke access at any time.
+
+
+== Creating a Connected App
+
+To create a Connected App, see xref:access-management::connected-apps-developers.adoc[].
+
+When you create the connected app to manage client applications, ensure that you:
+
+. Select *App acts on its own behalf (client credentials)* for *Type*.
+. Add the *Application Creator* scope.
+
+The new connected app can now create client applications and manage the client applications that it creates.
+
+
+== Creating a Client Application
+
+When a Connected App creates a client application, the owner of the Connected App is the owner of the connected App and the client application is created in the root organization. The Connected App receives the *Manage clients* scope for client application it creates. This is not visible the Connected App scopes list. 
```

Comment:
the owner of the "Connected App" is the owner of the connected App ---> the owner of the "Client App" is the owner of the connected App.

Reply:
Nit pick: Add "in" . This is not visible "in" the Connected App scopes list.